manifests/unit_file.pp

```diff
@@ -58,6 +58,13 @@
 # @param selinux_ignore_defaults
 #   maps to the same param on the file resource for the unit. false in the module because it's false in the file resource type
 #
+# @example manage unit file + service
+#   systemd::unit_file { 'foo.service':
+#     source => "puppet:///modules/${module_name}/foo.service",
```

Comment:
Personally I'd always use `content => file("${module_name}/foo.service")` but I suppose both work.

Reply:
ah yeah, I usually do that as well. fixed it.